src/api/membersApi.ts

```diff
@@ -0,0 +1,8 @@
+import { MemberType } from '../model/Member';
+import { getSupabaseClient } from './supabase';
+
+export const getMembers = async () => {
+  const { data } = await getSupabaseClient().from('members').select('*');
+
+  return data as MemberType[];
```

Comment:
supabase is very well done in terms of typescript support and ALSO! generating types automatically -> https://supabase.com/docs/reference/javascript/typescript-support

For the first few tables, we don't mind doing it by hand, for now. We can leverage the generic params of the select:

```suggestion
  const { data } = await getSupabaseClient().from('members').select<string, MemberType>();

  return data
```

and that will enable us to get rid of the explicit casting